twake/backend/node/src/services/channels/index.ts

```diff
@@ -28,6 +33,18 @@ export default class ChannelService extends TwakeService<ChannelServiceAPI> {
       next();
     });
 
+    /**
+     * Register private calls from php
+     */
+    phpnode.register({
```

Comment:
You still need to avoid to call phpnode from here but phpnode can call channels.